db/seeds.rb

```diff
@@ -5,3 +5,55 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+CITIES = [
+  {
+    city: "Boston",
+    state: "MA",
+    user_id: User.first.id
+  },
```

Comment:
Any reason you're setting up Boston twice? Just wanna make sure this doesn't cause issues for you down the line!